### ISSUE TITLE
migrate complains about changes not in migrations

### ISSUE THREAD

**Issue by santagada** [django-waffle/django-waffle#198]
running ./manage.py migrate it complains:
"Your models have changes that are not yet reflected in a migration, and so won't be applied.
  Run 'manage.py makemigrations' to make new migrations, and then re-run 'manage.py migrate' to apply them."

if you run makemigrations it generates:

```
# -*- coding: utf-8 -*-
# Generated by Django 1.9.5 on 2016-04-07 21:36
from __future__ import unicode_literals

from django.conf import settings
from django.db import migrations, models
import django.utils.timezone


class Migration(migrations.Migration):

    dependencies = [
        ('waffle', '0001_initial'),
    ]

    operations = [
        migrations.AlterField(
            model_name='flag',
            name='authenticated',
            field=models.BooleanField(default=False, help_text='Flag always active for authenticate users?'),
        ),
        migrations.AlterField(
            model_name='flag',
            name='created',
            field=models.DateTimeField(db_index=True, default=django.utils.timezone.now, help_text='Date when this Flag was created.'),
        ),
        migrations.AlterField(
            model_name='flag',
            name='everyone',
            field=models.NullBooleanField(help_text='Flip this flag on (Yes) or off (No) for everyone, overriding all other settings. Leave as Unknown to use normally.'),
        ),
        migrations.AlterField(
            model_name='flag',
            name='groups',
            field=models.ManyToManyField(blank=True, help_text='Activate this flag for these user groups.', to='auth.Group'),
        ),
        migrations.AlterField(
            model_name='flag',
            name='languages',
            field=models.TextField(blank=True, default='', help_text='Activate this flag for users with one of these languages (comma separated list)'),
        ),
        migrations.AlterField(
            model_name='flag',
            name='modified',
            field=models.DateTimeField(default=django.utils.timezone.now, help_text='Date when this Flag was last modified.'),
        ),
        migrations.AlterField(
            model_name='flag',
            name='name',
            field=models.CharField(help_text='The human/computer readable name.', max_length=100, unique=True),
        ),
        migrations.AlterField(
            model_name='flag',
            name='note',
            field=models.TextField(blank=True, help_text='Note where this Flag is used.'),
        ),
        migrations.AlterField(
            model_name='flag',
            name='percent',
            field=models.DecimalField(blank=True, decimal_places=1, help_text='A number between 0.0 and 99.9 to indicate a percentage of users for whom this flag will be active.', max_digits=3, null=True),
        ),
        migrations.AlterField(
            model_name='flag',
            name='rollout',
            field=models.BooleanField(default=False, help_text='Activate roll-out mode?'),
        ),
        migrations.AlterField(
            model_name='flag',
            name='staff',
            field=models.BooleanField(default=False, help_text='Flag always active for staff?'),
        ),
        migrations.AlterField(
            model_name='flag',
            name='superusers',
            field=models.BooleanField(default=True, help_text='Flag always active for superusers?'),
        ),
        migrations.AlterField(
            model_name='flag',
            name='testing',
            field=models.BooleanField(default=False, help_text='Allow this flag to be set for a session for user testing.'),
        ),
        migrations.AlterField(
            model_name='flag',
            name='users',
            field=models.ManyToManyField(blank=True, help_text='Activate this flag for these users.', to=settings.AUTH_USER_MODEL),
        ),
        migrations.AlterField(
            model_name='sample',
            name='created',
            field=models.DateTimeField(db_index=True, default=django.utils.timezone.now, help_text='Date when this Sample was created.'),
        ),
        migrations.AlterField(
            model_name='sample',
            name='modified',
            field=models.DateTimeField(default=django.utils.timezone.now, help_text='Date when this Sample was last modified.'),
        ),
        migrations.AlterField(
            model_name='sample',
            name='name',
            field=models.CharField(help_text='The human/computer readable name.', max_length=100, unique=True),
        ),
        migrations.AlterField(
            model_name='sample',
            name='note',
            field=models.TextField(blank=True, help_text='Note where this Sample is used.'),
        ),
        migrations.AlterField(
            model_name='sample',
            name='percent',
            field=models.DecimalField(decimal_places=1, help_text='A number between 0.0 and 100.0 to indicate a percentage of the time this sample will be active.', max_digits=4),
        ),
        migrations.AlterField(
            model_name='switch',
            name='active',
            field=models.BooleanField(default=False, help_text='Is this flag active?'),
        ),
        migrations.AlterField(
            model_name='switch',
            name='created',
            field=models.DateTimeField(db_index=True, default=django.utils.timezone.now, help_text='Date when this Switch was created.'),
        ),
        migrations.AlterField(
            model_name='switch',
            name='modified',
            field=models.DateTimeField(default=django.utils.timezone.now, help_text='Date when this Switch was last modified.'),
        ),
        migrations.AlterField(
            model_name='switch',
            name='name',
            field=models.CharField(help_text='The human/computer readable name.', max_length=100, unique=True),
        ),
        migrations.AlterField(
            model_name='switch',
            name='note',
            field=models.TextField(blank=True, help_text='Note where this Switch is used.'),
        ),
    ]
```

**Comment by jsocol** [comment]
This was probably solved in #178, I'm pushing a new version with that fix today.

**Comment by jsocol** [comment]
v0.11.1 is out and has the fix from #178. If that doesn't fix the issue, please re-open this!
